x-pack/plugins/security_solution/cypress/screens/alerts_detection_rules.ts

```diff
@@ -17,6 +17,12 @@ export const DELETE_RULE_ACTION_BTN = '[data-test-subj="deleteRuleAction"]';
 
 export const EDIT_RULE_ACTION_BTN = '[data-test-subj="editRuleAction"]';
 
+export const DUPLICATE_RULE_ACTION_BTN = '[data-test-subj="duplicateRuleAction"]';
+
+export const DUPLICATE_RULE_MENU_PANEL_BTN = '[data-test-subj="rules-details-duplicate-rule"]';
```

Comment:
I still go back and forth with the `data-test-subj` names. I try to follow https://github.com/elastic/kibana/blob/7.10/STYLEGUIDE.md#camel-case-id-and-data-test-subj but then see it differently in a lot of places.

Reply:
Yeah, for now, that one was not added by me, but I will from this point forward follow that rule. Thanks for pointing it out.

Really, we just need a linter rule and a linter fix for it.